src/mrack/logging.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Logging interface."""
+import logging
+
+DEBUG = logging.DEBUG
```

Comment:
I guess you are defining this so other parts can import that from mrack.logging, right?

How come the linter is not complaining about this?

Reply:
Really, dunno :D maybe because it is upper-case.

Reply:
This is not relevant anymore marking as resolved